scripts/quests/jeuno/LB09_2_Prelude_to_Puissance.lua

```diff
@@ -0,0 +1,109 @@
+-----------------------------------
+-- Prelude_to_Puissance
+-----------------------------------
+-- Log ID: 3, Quest ID: 170
+-- Nomad Moogle : !pos 10.012 1.453 121.883 243
+-----------------------------------
+require("scripts/settings/main")
+require("scripts/globals/items")
+require("scripts/globals/keyitems")
+require("scripts/globals/npc_util")
+require("scripts/globals/quests")
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local quest = Quest:new(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.PRELUDE_TO_PUISSANCE)
+-----------------------------------
+
+-- TODO: Properly code timing minigame. Awaiting for a capture.
+-- Amount of visual qeues selected at random. Min: Probably 3. Max: 7. Camera angle keeps changing qithout hints.
+-- Always the same camera angle at victory.
+
+quest.reward =
+{
+    fame  = 50,
+    fameArea = JEUNO,
+    keyItem = xi.ki.SOUL_GEM_CLASP,
+}
+
+quest.sections =
+{
+    -- Section: Quest available.
+    {
+        check = function(player, status)
+            return status == QUEST_AVAILABLE and
+                player:getMainLvl() >= 91 and
+                player:getLevelCap() == 95 and
+                xi.settings.MAX_LEVEL >= 99
+        end,
+
+        [xi.zone.RULUDE_GARDENS] =
+        {
+            ['Nomad_Moogle'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(10194)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [10194] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    -- Section: Quest accepted.
+    {
+        check = function(player, status)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.RULUDE_GARDENS] =
+        {
+            ['Nomad_Moogle'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:event(10045, 0, 1, 6, 2)
+                end,
+
+                onTrade = function(player, npc, trade)
+                    if npcUtil.tradeHasExactly(trade, xi.items.SEASONING_STONE) then
+                        return quest:progressEvent(10045, 0, 1, 5)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [10045] = function(player, csid, option, npc)
+                    -- All this options complete the trade and complete the current quest.
+                    if option == 13 or option == 14 or option == 15 or option == 19 or option == 20 or option == 21 then
+                        if quest:complete(player) then
+                            player:tradeComplete()
+
+                            -- This options immediately start next quest. (All except 15).
+                            if option == 13 or option == 14 or option == 19 or option == 20 or option == 21 then
```

Comment:
This can be simplified to `if option ~= 15` since the other values are required before entering this block